diff --git a/node_modules/immutability-helper/index.d.ts b/node_modules/immutability-helper/index.d.ts
index 3909c3d..942fae1 100644
--- a/node_modules/immutability-helper/index.d.ts
+++ b/node_modules/immutability-helper/index.d.ts
@@ -9,6 +9,7 @@ export declare const isEquals: any;
 export declare const extend: <T>(directive: string, fn: (param: any, old: T) => T) => void;
 declare const _default: <T, C extends CustomCommands<object> = never>(object: T, $spec: Spec<T, C>) => T;
 export default _default;
+export = _default;
 export declare type CustomCommands<T> = T & {
     __noInferenceCustomCommandsBrand: any;
 };
