diff --git a/node_modules/obs-websocket-js-4/types/index.d.ts b/node_modules/obs-websocket-js-4/types/index.d.ts
index de8d1da..904ca48 100644
--- a/node_modules/obs-websocket-js-4/types/index.d.ts
+++ b/node_modules/obs-websocket-js-4/types/index.d.ts
@@ -1,7 +1,7 @@
 // This file is generated, do not edit.
 // TypeScript Version: 3.1
 /// <reference types="node" />
-declare module "obs-websocket-js" {
+declare module "obs-websocket-js-4" {
   import { EventEmitter } from "events";
 
   namespace ObsWebSocket {
@@ -95,7 +95,7 @@ declare module "obs-websocket-js" {
     }
   }
 
-  interface RequestMethodsArgsMap {
+  export interface RequestMethodsArgsMap {
     GetVersion: void;
 
     GetAuthRequired: void;
@@ -550,7 +550,7 @@ declare module "obs-websocket-js" {
     StopVirtualCam: void;
   }
 
-  interface RequestMethodReturnMap {
+  export interface RequestMethodReturnMap {
     GetVersion: {
       messageId: string;
       status: "ok";
@@ -1451,5 +1451,5 @@ declare module "obs-websocket-js" {
     ): this;
   }
 
-  export = ObsWebSocket;
+  export default ObsWebSocket;
 }
